src/BaselineOfAIEditDistances/BaselineOfAIEditDistances.class.st

```diff
@@ -16,6 +16,9 @@ BaselineOfAIEditDistances >> baseline: spec [
 		spec
     		baseline: 'AIExternalVectorMatrix'
 			with: [ spec repository: 'github://pharo-ai/external-dependencies' ].
+		spec 
+			baseline: 'ContainersArray2D' 
+			with: [ spec repository: 'github://pharo-containers/Containers-Array2D/src' ].
 		"Packages"
 		spec
 			package: 'AI-EditDistances' with: [ spec requires: #( 'AIExternalVectorMatrix' ) ];
```

Comment:
you need to add the dependency to ContainersArray2D here, just as external vector matrix